ister_gui.py

```diff
@@ -910,7 +995,7 @@ def __init__(self):
     def get_next_step(self, action):
         """Returns the process step matched by action"""
         if action == 'Exit':
-            exit(0)
+            sys.exit(0)
```

Comment:
These changes to exit being called confuse me a little. Mind explaining this one?

Reply:
exit(0) is a helper for the python interactive session and sys.exit(0) is what must be used for scripts